Tweak modlist

D-Alessian opened the pull request:
NEED people to review this. We need to make sure to remove useless libs and that I didn't remove important mods. 
We need to triple check this

D-Alessian committed:
1st iteration of a new modlist

D-Alessian committed:
sync manifests

D-Alessian committed:
add ZBGT, OC, AE2FCr. Remove Thermal. Fix a print to console left for debug.

D-Alessian committed:
Merge remote-tracking branch 'origin/master' into tweak-modlist
merge upstream

D-Alessian committed:
forgot to merge from master, use the better manifest scripts to sync and use the better names in the template

D-Alessian committed:
merge upstream

D-Alessian committed:
Remove forgotten enderio and thermal based mods, and add packmode back

D-Alessian committed:
Remove redstone arsenal and CoFH

D-Alessian committed:
Add cfgs from Nomi-CEu and remove (hopefully) every deleted mod's config

D-Alessian committed:
goofy ahh solar flux forgor to comment out their code and it made my IDE error

D-Alessian committed:
change the packmode to expert (temp solution for packmode)

D-Alessian committed:
change the bg color for loading the pack to red

D-Alessian committed:
forgot to delete AE2WT's cfg

D-Alessian committed:
change Nomi Lab's configs for HOGi

D-Alessian committed:
add Nomi CEu's resources without mods we don't have (hopefully)

D-Alessian committed:
remove .gitkeep in overrides

D-Alessian committed:
bro... solar flux what are you doing.................

This reverts commit d306724a802a907aa587690a940787f58fd7f832.

smallming675 reviewed (commented):
game successfully launched, it works

D-Alessian reviewed (commented):
assets to change: 
- `overrides/config/betterquesting/resources/betterquesting/textures/gui/default_title.png`
- `overrides/resources/minecraft/textures/gui/title/background/*`
- `overrides/resources/minecraft/textures/gui/title/*`
- `overrides/resources/modpack/textures/gui/default_title_expert.png`
- `overrides/resources/modpack/textures/gui/default_title_normal.png`
- `overrides/resources/modpack/textures/logo/*`

D-Alessian committed:
add our logo to the textures

D-Alessian committed:
Easy commit for build actions

Signed-off-by: Alessian Dormal <alessian@alessian.be>

D-Alessian committed:
B
Merge branch 'master' of https://github.com/Horde-Of-Greg/HOGifactory into tweak-modlist
merge upstream

D-Alessian committed:
Revert manifest.json to original state for actions, and add auto versioning to it

D-Alessian committed:
Reverted manifest.json wrong, left a test change in syncMainManifest.js, and actions were wrong for author and user being switched around

Commit:
sync manifest.json from template

D-Alessian committed:
Just refactoring the version bump logic to avoid a possible scenario where github would give a bump on every commit

D-Alessian committed:
This action was a bad idea it won't work properly. Will impl some check for it, but later.

D-Alessian committed:
Me when I forgret to change template code

D-Alessian committed:
Start implementing review comments. Cooking a main menu

D-Alessian committed:
Forgot to put the version bump in tools as working directory

D-Alessian committed:
There is something wrong with me

Commit:
sync manifest.json from template

D-Alessian committed:
2 stupid mistakes

D-Alessian committed:
Merge remote-tracking branch 'origin/tweak-modlist' into tweak-modlist

Commit:
sync manifest.json from template

D-Alessian committed:
Improve main menu

D-Alessian committed:
Merge remote-tracking branch 'origin/tweak-modlist' into tweak-modlist

Commit:
sync manifest.json from template

D-Alessian committed:
Make almost all other requested changes

D-Alessian committed:
Merge remote-tracking branch 'origin/tweak-modlist' into tweak-modlist

Commit:
sync manifest.json from template

D-Alessian committed:
Improve main menu

Commit:
sync manifest.json from template

D-Alessian committed:
add, probably, a better way to check if sync manifest changed anything

Commit:
sync manifest.json from template

D-Alessian committed:
I read this might solve the issue? Idk atp

D-Alessian committed:
Merge branch 'tweak-modlist' of github.com:Horde-Of-Greg/HOGifactory into tweak-modlist
merge upstream

Commit:
sync manifest.json from template

D-Alessian committed:
Just add a debug output to console

Commit:
sync manifest.json from template

D-Alessian committed:
Another debug string to understand??????

D-Alessian committed:
Merge branch 'tweak-modlist' of github.com:Horde-Of-Greg/HOGifactory into tweak-modlist
ugh I keep having to merge upstream

Commit:
sync manifest.json from template

D-Alessian committed:
Oh my god I think this is the problem, why did this get here??

Commit:
sync manifest.json from template

D-Alessian committed:
Im literally so stupid I wasnt changing the template, it should be fine now??

D-Alessian committed:
ahhh typo

D-Alessian committed:
I cant stop being stupid today. But also, go back to using cmp

Commit:
sync manifest.json from template

D-Alessian committed:
I wasnt in the right dir when comitting, what am I doing

Commit:
sync manifest.json from template

D-Alessian committed:
And now Im stupid about this too

D-Alessian committed:
Remove prints to console used for testing
